Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -334,6 +334,9 @@ theorem preAleph_omega0 : preAleph ω = ℵ₀ :=
 theorem aleph0_le_preAleph {o : Ordinal} : ℵ₀ ≤ preAleph o ↔ ω ≤ o := by
   rw [← preAleph_omega0, preAleph_le_preAleph]
 
+theorem range_preAleph : range preAleph = Set.univ :=
```

Comment:
This one doesn't seem that necessary to me - `simp` already knows it, and `rw?` gets it too, the specialisation here appears unhelpful.

Reply:
I only added it for symmetry with `range_aleph`, but I don't really mind removing it.

Reply:
Yeah let's get rid of it - `range_aleph` isn't immediately deduced from something else whereas this one is